providers/fab/tests/unit/fab/www/views/test_views_custom_user_views.py

```diff
@@ -26,7 +26,7 @@
 
 from airflow import settings
 from airflow.providers.fab.www import app as application
-from airflow.security import permissions
+from airflow.providers.fab.www.security import permissions
```

Comment:
Not especially related. I can create a separate PR just for this if you prefer. Permissions are duplicated in core airflow and FAB provider (ideally it should be removed from core Airflow but that requires a bit of work). In FAB provider, let's use permissions from FAB provider